### PR TITLE
Crashing on tab switches bug fix

### PR DESCRIPTION
**Pull request by SunnyNagam**

### CODE REVIEW

File: app/src/main/java/com/cobresun/factfulnewsandroid/TabFragment.kt [hunk 1]
```diff
@@ -3,12 +3,12 @@ package com.cobresun.factfulnewsandroid
 
 import android.net.Uri
 import android.os.Bundle
-import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import androidx.browser.customtabs.CustomTabsIntent
 import androidx.core.content.res.ResourcesCompat
+import androidx.fragment.app.Fragment
```
Comment (wesley-tang) [inline app/src/main/java/com/cobresun/factfulnewsandroid/TabFragment.kt:12]: Did you move where this import was? LMAO

**Review by wesley-tang — APPROVED** [review]
Remember that this is a HOTFIX and that a proper solution will need to be implemented in the future. Create a ticket for it perhaps?

File: app/src/main/java/com/cobresun/factfulnewsandroid/TabFragment.kt [hunk 2]
```diff
@@ -38,13 +37,15 @@ class TabFragment(index: Int) : Fragment() {
         val api = retrofit.create(ApiService::class.java)
         api.fetchArticles(CategoryUtils.categories[tabIndex]).enqueue(object: Callback<FetchResponse> {
             override fun onResponse(call: Call<FetchResponse>, response: Response<FetchResponse>) {
-                showArticles(response.body()!!.articles)
+                if(activity != null)
+                    showArticles(response.body()!!.articles)
```
Comment (brian-norman) [inline app/src/main/java/com/cobresun/factfulnewsandroid/TabFragment.kt]: Should be able to get rid of the "!!" if we want to be right about this !!

File: app/src/main/java/com/cobresun/factfulnewsandroid/ui/activities/MainActivity.kt [hunk 3]
```diff
@@ -31,10 +31,7 @@ class MainActivity : AppCompatActivity() {
         // Each page is represented by its own fragment.
         val viewPager = findViewById<ViewPager>(R.id.pager)
 
-        var adapter: TabsPagerAdapter
-        adapter = TabsPagerAdapter(supportFragmentManager, tabLayout.tabCount)
-
-        viewPager.adapter = adapter
+        viewPager.adapter = TabsPagerAdapter(supportFragmentManager, tabLayout.tabCount)
```
Comment (brian-norman) [inline app/src/main/java/com/cobresun/factfulnewsandroid/ui/activities/MainActivity.kt:34]: Nice clean :+1:

**Review by brian-norman — CHANGES_REQUESTED** [review]

**Review by brian-norman — APPROVED** [review]

File: app/src/main/java/com/cobresun/factfulnewsandroid/TabFragment.kt (hunk 1 above)
Comment (SunnyNagam) [reply app/src/main/java/com/cobresun/factfulnewsandroid/TabFragment.kt:12]: oh yea idk why

**Pull request merged**